secantTests/Util/KeychainManagerTests.swift

```diff
@@ -0,0 +1,256 @@
+//
+//  KeychainManagerTests.swift
+//  secantTests
+//
+//  Created by Lukáš Korba on 10.03.2022.
+//
+
+import XCTest
+@testable import secant_testnet
+
+class KeychainManagerTests: XCTestCase {
+    // MARK: - Seed Phrase tests
+    
+    var storage: RecoveryPhraseStorage?
+    let seedPhraseSet = "one two three"
+    let birthdaySet = BlockHeight(12345678)
+
+    override func setUp() {
+        super.setUp()
+        storage = RecoveryPhraseStorage()
+        deleteData(forKey: "zcashSeedPhrase")
+        deleteData(forKey: "zcashBirthday")
+    }
+    
+    override func tearDown() {
+        storage = nil
+        super.tearDown()
```

Comment:
It would be more typical to call super prior to executing any other functionality right?